Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.Properties.cs

```diff
@@ -154,14 +164,10 @@ public ICommand ItemClickCommand
         }
 
         /// <summary>
-        /// Gets or sets the selected multiple objects in the collection.
+        /// Gets the selected multiple objects in the collection.
         /// </summary>
-        /// <value>The object that is used to store selected multiple items. Default is an empty IList collection of object.</value>
-        public IList<object> SelectedItems
-        {
-            get { return (IList<object>)GetValue(SelectedItemsProperty); }
-            set { SetValue(SelectedItemsProperty, value); }
-        }
+        /// <value>The object that is used to store selected multiple items.</value>
+        public IList<object> SelectedItems { get; }
```

Comment:
Well, missed a few lines there. This actually breaks the xaml and will be fixed in a few minutes. Thanks for pointing out!